### PR TITLE
Add OTC to wallet

### PR DESCRIPTION
**Pull request by montycrypto**
This PR adds support for OTC (aimed at event.skycoin.net) to the wallet.

Due to event.skycoin.net being offline (and not having CORS-headers), the current dist files will not work. You can however test by running skycoin/teller locally and using the available proxy on localhost:4200/teller.

### CODE REVIEW

**Comment by gz-c** [comment]
Hide all behind feature flag until ready (need the wallet frontend to remain in release-ready state)

**Comment by montycrypto** [comment]
Feature flag has been added to a config file (app.config.ts), also changed Hours to Coin Hours (closes #526)

File: src/gui/static/README.MD [hunk 1]
```diff
@@ -27,3 +27,11 @@ Please note that you will most likely receive CORS errors as there's a differenc
...
 
 You can route all calls to this address by changing the url property on the ApiService class.
+
+## Purchase API (teller)
+
+Please note that at the moment the Purchase API (teller) is both offline and not supporting CORS headers.
+
+While event.skycoin.net is not working, we will have to run the purchase API locally.
+
+Similar as the solution for the above CORS issue, you can circumvent CORS issues by chaging the url property to '/teller/'
```
Comment (gz-c) [inline src/gui/static/README.MD]: "chaging" -> "changing"

File: src/gui/static/src/app/components/pages/buy/buy.component.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<p *ngIf="!otcEnabled">Sorry, otc has currently been disabled!</p>
+<md-card *ngIf="otcEnabled">
+  <md-card-title>Purchase Skycoin</md-card-title>
+  <md-card class="skycoin-details">
+    You can buy Skycoins directly from your wallet using our Skycoin Teller service. The current rate is 0.0002 BTC per
```
Comment (gz-c) [inline src/gui/static/src/app/components/pages/buy/buy.component.html]: Will need to show a dynamic rate once we add that api to teller

File: src/gui/static/src/app/pipes/teller-status.pipe.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+@Pipe({
+  name: 'tellerStatus'
+})
+export class TellerStatusPipe implements PipeTransform {
+
+  transform(value: any): any {
+    switch (value) {
+      case 'waiting_deposit':
+        return 'Waiting for deposit';
```
Comment (gz-c) [inline src/gui/static/src/app/pipes/teller-status.pipe.ts]: Waiting for Bitcoin deposit

File: src/gui/static/src/app/pipes/teller-status.pipe.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      case 'waiting_send':
+        return 'Waiting for sending out';
```
Comment (gz-c) [inline src/gui/static/src/app/pipes/teller-status.pipe.ts]: Waiting to send Skycoin

File: src/gui/static/src/app/components/pages/buy/buy.component.html (hunk 2 above)
Comment (montycrypto) [reply src/gui/static/src/app/components/pages/buy/buy.component.html]: Very true, I have opened an issue for this on the teller project: https://github.com/skycoin/teller/issues/54

File: src/gui/static/README.MD (hunk 1 above)
Comment (montycrypto) [reply src/gui/static/README.MD]: Done

File: src/gui/static/src/app/pipes/teller-status.pipe.ts (hunk 3 above)
Comment (montycrypto) [reply src/gui/static/src/app/pipes/teller-status.pipe.ts]: Done

File: src/gui/static/src/app/pipes/teller-status.pipe.ts (hunk 4 above)
Comment (montycrypto) [reply src/gui/static/src/app/pipes/teller-status.pipe.ts]: Done

**Pull request merged**